### PR TITLE
Clever sticky footer

### LINKED ISSUE #1008: Documentation footer is not sticky
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

On a big screen, when we go to "Design guidelines", the footer is not sicky and is display at around 3/4 of the screen.

Screenshot from 2021-12-30 07-50-45

### Reduced test cases

At the time of writing this, it can only be seen locally at http://localhost:9001/docs/5.1/guidelines/ since it will be deployed with the 5.2.0 version of Boosted.

Maybe worth a try: [A Clever Sticky Footer Technique](https://css-tricks.com/a-clever-sticky-footer-technique/).

Note: even if we don't see it in Bootstrap since there is not short pages there, this issue exists as well.

### What operating system(s) are you seeing the problem on?

Linux

### What browser(s) are you seeing the problem on?

Firefox

### What version of Boosted are you using?

v5.2.0 (current _main_ branch)

### PR DESCRIPTION
**Pull request by MewenLeHo**
Closes #1008 

- [x]  I have searched for duplicate or closed issues
- [x]  I have validated any HTML to avoid common problems
- [x]  I have read the contributing guidelines

### CODE REVIEW

File: site/assets/scss/_boosted.scss [hunk 1]
```diff
@@ -15,6 +15,11 @@ body {
   }
 }
 
+html,
+body {
+  min-height: 100%;
```
Comment (louismaximepiton) [inline site/assets/scss/_boosted.scss:20]: It works best with `min-height: 100vh` here :

100% : 
image

100vh :

**Comment by julien-deramond** [comment]
FYI, there's an issue with the rebase:

Screenshot from 2022-04-08 11-50-56

**Comment by julien-deramond** [comment]
There's in an issue in the "Orange Design System for web" page.
The scrollbar appears when the page is loaded:

Screenshot from 2022-04-08 12-16-02

When we scroll until the end of the page, the back to top button is not where it is supposed to be.

Screenshot from 2022-04-08 12-16-19

But apparently the issue is not maybe related to this modification given the result displayed in https://boosted.netlify.app/docs/5.1/guidelines/.

**Comment by louismaximepiton** [comment]
IMO, this issue is caused by #765. So this one could be merged without fixing it !

**Review by julien-deramond — APPROVED** [review]
Let's create an issue after the merge of this one to tackle the back-to-top problem.

**Pull request merged**